pkg/conc/ddb_lock_provider_test.go

```diff
@@ -41,16 +60,15 @@ func (s *ddbLockProviderTestSuite) SetupTest() {
 	logger := monMocks.NewLoggerMockedAll()
 	s.ctx = context.Background()
 	s.repo = new(ddbMocks.Repository)
-	s.clock = clockwork.NewFakeClock()
+	s.clock = clock.NewFakeClock()
 	s.uuidSource = new(uuidMocks.Uuid)
 	s.uuidSource.On("NewV4").Return(s.token).Once()
-	s.backOff = &backoff.ExponentialBackOff{
-		InitialInterval:     time.Millisecond,
-		RandomizationFactor: 0,
-		Multiplier:          2,
-		MaxInterval:         time.Millisecond * 4,
-		MaxElapsedTime:      time.Millisecond * 8,
-		Clock:               clockwork.NewRealClock(), // use a real clock, otherwise time does not advance without us
+	s.backOff = &testBackOff{
+		backOffs: []time.Duration{
+			time.Millisecond * 1,
+			time.Millisecond * 2,
+			time.Millisecond * 4,
```

Comment:
any reason these should be different to one another or can we just add time.Millisecond without a multiplicator?

Reply:
Because that is what the real code would wait for and I wanted a test to resemble the real situation as closely as possible (although the real value is most likely irrelevant anyway). I can change it if you want.